language-support/hs/bindings/src/DA/Ledger.hs

```diff
@@ -34,8 +35,11 @@ import DA.Ledger.Types
 import UnliftIO (liftIO,timeout,bracket)
 
 configOfPort :: Port -> ClientConfig
-configOfPort port =
-    ClientConfig { clientServerHost = Host "localhost"
+configOfPort = configOfHostAndPort "localhost"
+
+configOfHostAndPort :: Host -> Port -> ClientConfig
+configOfHostAndPort host port =
```

Comment:
I would probably just inline the definition at this point. The helper doesn’t add that much. But I don’t feel strongly about this so your choice.